src/unix/bsd/netbsdlike/openbsd/mod.rs

```diff
@@ -1604,6 +1608,20 @@ extern "C" {
     pub fn pthread_get_name_np(tid: ::pthread_t, name: *mut ::c_char, len: ::size_t);
     pub fn pthread_set_name_np(tid: ::pthread_t, name: *const ::c_char);
     pub fn pthread_stackseg_np(thread: ::pthread_t, sinfo: *mut ::stack_t) -> ::c_int;
+    pub fn pthread_setschedparam(
+        native: ::pthread_t,
+        policy: ::c_int,
+        param: *const sched_param,
+    ) -> ::c_int;
+    pub fn pthread_getschedparam(
+        native: ::pthread_t,
+        policy: *mut ::c_int,
+        param: *mut sched_param,
+    ) -> ::c_int;
+    pub fn sched_setparam(pid: ::pid_t, param: *const sched_param) -> ::c_int;
+    pub fn sched_getparam(pid: ::pid_t, param: *mut sched_param) -> ::c_int;
+    pub fn sched_getscheduler(pid: ::pid_t) -> ::c_int;
+    pub fn sched_setscheduler(pid: ::pid_t, policy: ::c_int, param: *const sched_param) -> ::c_int;
```

Comment:
the 4 `sched_*` functions aren't defined on OpenBSD. In fact, they exists in [sched.h](https://github.com/openbsd/src/blob/master/include/sched.h#L57) but under `#if 0`. so it will not link.

Reply:
Oh no! :( Thanks for noticing this. Really love that "not yet" comment there.

Reply:
I've removed those.